cmake/platform.cmake

```diff
@@ -0,0 +1,264 @@
+
+
+###
+### Platform flags
+### TODO: arch flags. Defaulting to x86-64
+
+message(STATUS "CMAKE_SYSTEM_NAME=${CMAKE_SYSTEM}")
+message(STATUS "CMAKE_SYSTEM_VERSION=${CMAKE_SYSTEM_VERSION}")
+message(STATUS "CMAKE_SYSTEM_PROCESSOR=${CMAKE_SYSTEM_PROCESSOR}")
+message(STATUS "CMAKE_CROSSCOMPILING=${CMAKE_CROSSCOMPILING}")
+
+#set(OMR_ARCH_POWER Off Internal "Power CPU Architecture")
```

Comment:
are these commented out statements the fundamental spot to enable platforms that aren't yet supported by our cmake build implementation? If so, please leave a comment to that effect. If it's not, please leave a comment explaining what they are for.